app/controllers/api/solutions/submission_test_runs_controller.rb

```diff
@@ -10,9 +10,12 @@ def show
 
         render json: {
           test_run: test_run,
-          test_runner_status: {
-            exercise: submission.exercise.has_test_runner,
-            track: submission.track.has_test_runner
+          test_runner: {
+            average_test_duration: submission.track.average_test_duration,
+            status: {
+              exercise: submission.exercise.has_test_runner,
+              track: submission.track.has_test_runner
+            }
```

Comment:
I now also need to pass in the average test duration. I could have added it as just a new field, but I felt that encapsulating it in a new, top-level `test_runner` field made more sense.